apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaClaimMessageMetaData.java

```diff
@@ -82,10 +82,10 @@ public class RdaClaimMessageMetaData {
   /**
    * Timestamp when we inserted the claim from this message into our database. We might have
    * multiple records for the same message in our database if we replay old data from the RDA API
```

Comment:
I think we can delete the second sentence.  Sequence number should be unique now, so if we reply data we'll now UPDATE the existing record in this table instead of adding a new row.